atlasdb-jepsen-tests/src/jepsen/atlasdb.clj

```diff
@@ -1,178 +1,99 @@
-   (c/upload "resources/atlasdb/atlasdb-ete.yml.template" "/atlasdb-ete/var/conf/")
-   (c/cd
-    "/atlasdb-ete"
-    (c/exec :sed (format "s/{{LOCAL_HOSTNAME}}/%s/g" (name node)) "var/conf/atlasdb-ete.yml.template" :> "var/conf/atlasdb-ete.yml")
-    (c/exec "service/bin/init.sh" "start"))
-   (c/exec (c/lit (format "(false; while [[ $? != 0 ]]; do sleep 1; curl -sfo /dev/null http://%s:3828/cas; done)" (name node))))
-   (info node "AtlasDB is ready")))
-
-(defn install-cassandra
-  [node version]
-  (c/su
-   (info node "installing Cassandra" version)
-   (debian/add-repo!
-    "cassandra"
-    "deb http://www.apache.org/dist/cassandra/debian 22x main")
-   (debian/install {:cassandra version})
-   (info node "starting Cassandra")
-   (c/upload "resources/cassandra/cassandra.yaml.template" "/etc/cassandra/")
-   (c/exec :sed (c/lit "\"s/{{LOCAL_ADDRESS}}/$(hostname -I)/g\"") "/etc/cassandra/cassandra.yaml.template" :> "/etc/cassandra/cassandra.yaml")
-   (c/upload "resources/cassandra/cassandra-env.sh" "/etc/cassandra/")
-   (c/exec :service :cassandra :start)
-   (info node "Cassandra is ready")))
-
-(defn teardown-atlasdb
-  [node]
-  (c/su
-   (info node "tearing down AtlasDB")
-   (c/cd
-    "/atlasdb-ete"
-    (c/exec "service/bin/init.sh" "stop"))))
-
-(defn teardown-cassandra
-  [node]
-  (c/su
-   (info node "tearing down Cassandra")
-   (c/exec :service :cassandra :stop)
-   (c/exec :rm "-rf" "/var/lib/cassandra/{saved_caches,data,commitlog}")))
-
-(defn db
-  "AtlasDB node setup."
+            [jepsen.util :refer [timeout]]
+            [knossos.history :as history]
+            [jepsen.tests :as tests])
+  (:import com.palantir.atlasdb.jepsen.JepsenHistoryChecker)
+  (:import com.palantir.atlasdb.jepsen.TimestampClient))
+
+(defn create-server
+  "Creates an object that implements the db/DB protocol.
+   This object defines how to setup and teardown a timelock server on a given
+   node, and specifies where the log files can be found.
+  "
   []
   (reify db/DB
-    (setup! [_ test node]
-      (if (some #{node} (:nodes test))
-        (install-atlasdb node))
-      (if (some #{node} (:cassandra-nodes test))
-        (install-cassandra node CASSANDRA_VERSION)))
-
-    (teardown! [_ test node]
-      (if (some #{node} (:nodes test))
-        (teardown-atlasdb node))
-      (if (some #{node} (:cassandra-nodes test))
-        (teardown-cassandra node)))
+    (setup! [_ _ node]
+      (c/su
+        (debian/install-jdk8!)
+        (c/upload "resources/atlasdb/atlasdb-timelock-server.tgz" "/")
+        (info node "Uploading and unpacking timelock server")
+        (c/exec :mkdir "/atlasdb-timelock-server")
+        (c/exec :tar :xf "/atlasdb-timelock-server.tgz" "-C" "/atlasdb-timelock-server" "--strip-components" "1")
+        (c/upload "resources/atlasdb/timelock.yml" "/atlasdb-timelock-server/var/conf")
+        (c/exec :sed :-i (format "'s/HOSTNAME/%s/'" (name node)) "/atlasdb-timelock-server/var/conf/timelock.yml")
```

Comment:
Good idea: I've gone for `<HOSTNAME>`, since that played the nicest with the `c/exec :sed` line, and it can't be confused with it being an environment variable.